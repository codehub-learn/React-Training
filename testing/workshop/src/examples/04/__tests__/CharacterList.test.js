import { render } from "@testing-library/react";
import CharacterList from "../CharacterList";

describe("<CharacterList />", () => {
  it("renders correctly with snapshot", () => {
    const { container } = render(<CharacterList />);

    expect(container).toMatchSnapshot();
  });
});
