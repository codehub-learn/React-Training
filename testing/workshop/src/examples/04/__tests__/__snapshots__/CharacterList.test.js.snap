// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CharacterList /> renders correctly with snapshot 1`] = `
<div>
  <div>
    <h1>
      My title
    </h1>
    <hr />
    <div>
      <div
        class="entry"
      >
        <h2>
          John Tsevdos
        </h2>
        <p>
          male
        </p>
        <button
          type="primary"
        >
          edit 
          John Tsevdos
        </button>
      </div>
      <div
        class="entry"
      >
        <h2>
          Patricia Lebsack
        </h2>
        <p>
          female
        </p>
        <button
          type="primary"
        >
          edit 
          Patricia Lebsack
        </button>
      </div>
      <div
        class="entry"
      >
        <h2>
          Dennis Schulist
        </h2>
        <p>
          male
        </p>
        <button
          type="primary"
        >
          edit 
          Dennis Schulist
        </button>
      </div>
    </div>
  </div>
</div>
`;
